src/main/java/net/sf/jabref/gui/JabRefFrame.java

```diff
@@ -622,22 +624,26 @@ public void windowClosing(WindowEvent e) {
         tabbedPane.addChangeListener(e -> {
             markActiveBasePanel();
 
-            BasePanel bp = getCurrentBasePanel();
-            if (bp == null) {
+            BasePanel currentBasePanel = getCurrentBasePanel();
+            if (currentBasePanel == null) {
+                globalSearchBar.setVisible(false);
+                globalSearchBar.setAutoCompleter(null);
                 return;
             }
```

Comment:
This only disables the global search bar when the last database will be closed. But the search bar will be shown when one is starting jabref with no open database.